pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
Yes, I thought about that. Since there is no TRiP handler for these files anyway (perhaps except for SVV, but that could be added later anyway), I think the best would be to pickle it for saving.